pipeline/relation_embedder/src/test/scala/uk/ac/wellcome/relation_embedder/RelatedWorksServiceTest.scala

```diff
@@ -51,110 +51,182 @@ class RelatedWorksServiceTest
   val works =
     List(workA, workB, workC, workD, workE, workF, work4, work3, work2, work1)
```

Comment:
Maybe it's annoying, but i wonder if it might make the tests easier to read if the works were named
```scala
  val workA = work("a")
  val workA1 = work("a/1")
  val workA1B = work("a/1/b")
  val workA1C = work("a/1/c")
  val workA2 = work("a/2")
  val workA2D = work("a/2/d")
  val workA2E = work("a/2/e")
  val workA2EF = work("a/2/e/f")
  val workA3 = work("a/3")
  val workA4 = work("a/4")
```

Could just be because I'm seeing the changes in the PR out of context of where we set the vals